### DISCUSSION TITLE
layer-name怎么设置？

### DISCUSSION THREAD

**Discussion by weizmann** [Tencent/libpag#1674] [Q&A] [answered]
我看到api里有很多通过layer-name来操作的方法，但是不知道如何设置。
如：
`string PAGLayer::layerName()` / `getLayersByName()` / `replaceImageByName`

想问下如何为pag文件设置layer-name？

比如，下面这张图，用index管理不太友好，有没有设置layername的地方？
![image](https://github.com/Tencent/libpag/assets/2306111/f9bdd7b1-b461-44f1-a512-7adeadad0747)

**Comment by kevingpqi123** [comment]
让你们的设计师在 AE 工程中设置图层名称，桌面端预览工具不支持修改 LayerName

**Reply by 0x1306a94** [reply, marked as answer]
就是AE里面这里的图层名称 @weizmann 
![WX20230821-145340@2x](https://github.com/Tencent/libpag/assets/14822396/497777f1-cace-4c81-87c9-a8e998634fc0)
